explain-overview.md

```diff
@@ -6,500 +6,52 @@ aliases: ['/docs/dev/query-execution-plan/','/docs/dev/reference/performance/und
-```
-
-Generally speaking, `Hash Aggregate` is executed in two stages.
-
-- One is on the Coprocessor of TiKV/TiFlash, with the intermediate results of the aggregation function calculated when the table scan operator reads the data.
-- The other is at the TiDB layer, with the final result calculated through aggregating the intermediate results of all Coprocessor Tasks.
-
-The operator info column in the `explain` table also records other information about `Hash Aggregation`. You need to pay attention to what aggregate function that `Hash Aggregation` uses. In the above example, the operator info of the `Hash Aggregation` operator is `funcs:count(Column#7)->Column#4`. It means that `Hash Aggregation` uses the aggregate function `count` for calculation. The operator info of the `Stream Aggregation` operator in the following example is the same with this one.
-
-#### `Stream Aggregate` example
-
-The `Stream Aggregation` operator usually takes up less memory than `Hash Aggregate`. In some scenarios, `Stream Aggregation` executes faster than `Hash Aggregate`. In the case of a large amount of data or insufficient system memory, it is recommended to use the `Stream Aggregate` operator. An example is as follows:
-
-{{< copyable "sql" >}}
-
-```sql
-explain select /*+ STREAM_AGG() */ count(*) from t;
-```
-
-```sql
-+----------------------------+----------+-----------+---------------+---------------------------------+
-| id                         | estRows  | task      | access object | operator info                   |
-+----------------------------+----------+-----------+---------------+---------------------------------+
-| StreamAgg_16               | 1.00     | root      |               | funcs:count(Column#7)->Column#4 |
-| └─TableReader_17           | 1.00     | root      |               | data:StreamAgg_8                |
-|   └─StreamAgg_8            | 1.00     | cop[tikv] |               | funcs:count(1)->Column#7        |
-|     └─TableFullScan_13     | 10000.00 | cop[tikv] | table:t       | keep order:false, stats:pseudo  |
-+----------------------------+----------+-----------+---------------+---------------------------------+
-4 rows in set (0.00 sec)
-```
-
-Similar to `Hash Aggregate`, `Stream Aggregate` is executed in two stages.
-
-- One is on the Coprocessor of TiKV/TiFlash, with the intermediate results of the aggregation function calculated when the table scan operator reads the data.
-- The other is at the TiDB layer, with the final result calculated through aggregating the intermediate results of all Coprocessor Tasks.
-
-### Read the `Join` execution plan
-
-The `Join` algorithms in TiDB consist of the following categories:
-
-- [Hash Join](#hash-join-example)
-- [Merge Join](#merge-join-example)
-- [Index Join (Index Nested Loop Join)](#index-join-example)
-- [Index Hash Join (Index Nested Loop Hash Join)](#index-hash-join-example)
-- [Index Merge Join (Index Nested Loop Merge Join)](#index-merge-join-example)
```

Comment:
I think this kind of list enables users to easily find specific information. What about doing the same thing in each one of the `EXPLAIN` series (to improve retrievability) and listing this series in `explain-overview.md`?

Some of the `explain-*.md` docs do not give enough context at the beginning and do not introduce what these docs are about (not self-contained). What about telling readers that this is a series of docs and giving them `See also`?

Reply:
The explain series approaches it from a more logical perspective: joins -> then shows hash/merge/ index join. subqueries -> these are subqueries. I don't think approaching it straight from hash join makes sense.